ext/drcovlib/drcovlib.c

```diff
@@ -213,7 +213,7 @@ static void
 dump_drcov_data(void *drcontext, per_thread_t *data)
 {
     version_print(data->log);
-    module_table_print(module_table, data->log, false);
+    module_table_print(module_table, data->log, options.verbose_module_table);
```

Comment:
No, if you run drcov -verbose 1 today, everything is fine.
This CL will make -verbose dump some extra module table information into the output file. The post-processing may fail to read the module table, at least read wrong info.
xref https://github.com/DynamoRIO/dynamorio/blob/master/ext/drcovlib/modules.c#L197 and https://github.com/DynamoRIO/dynamorio/blob/master/clients/drcov/postprocess/drcov2lcov.cpp#L779 on module table info printing and reading.
So I am suggesting to make the printing/reading consistent.